### ISSUE TITLE
v5 source code?

### ISSUE THREAD

**Issue by tumatanquang** [javolution/javolution#47]
Hi, I would like to ask where can I download the v5 project, versions 5.3.x, 5.4.x, 5.5.x? Github doesn't seem to have these branches, and also doesn't have the build.xml file.

**Comment by dautelle** [comment]
Hello,
Unfortunately Javolution is not maintained anymore (I moved to something
else), unless someone wants to take it over. It will disappear.
Best regards,
Jean-Marie

**Comment by tumatanquang** [comment]
@dautelle It's sad to hear this even though this project still has a lot of users. However, I really hope to continue using and maintaining it. So I really hope that you can share with me the source code of this project, especially the v5 source code.

**Comment by dautelle** [comment]
Yes it is sad but it was a good experience which for me lasted 10 years (I
moved to the AI technical field now). Here is a link
<https://drive.google.com/file/d/1_MhZz5otxkxll4Om5vrcKLh5WdkiEtYn/view?usp=sharing>
to latest source code I have. Let me know if you or anyone is interested in
taking over, I can give you ownership rights on github.
Cheers,
Jean-Marie

**Comment by tumatanquang** [comment]
I checked the file you sent, it seems to be the source code of the current version (v6 or v7)? Can you share me the pom.xml and build.xml files of v5.3.x - 5.5.x and of the current version?

**Comment by dautelle** [comment]
Later versions of javolution were built and released using maven (and not
ant).
You can find the pom.xml on maven central, e.g. for version 6.0.0
https://repo1.maven.org/maven2/org/javolution/javolution/6.0.0/
Cheers,
jmd

**Comment by tumatanquang** [comment]
I tried rebuilding v5 but using the build.xml file found in [this project](https://github.com/javolution/javolution/tree/V5_2) on github. But it doesn't seem to be compatible with v5.3.x and above so it always fails to compile. I will try again with v6 later.

**Comment by tumatanquang** [comment]
I recompiled Javolution 5.5.1 directly from source code: [Javolution non-final](https://github.com/tumatanquang/javolution-non-final)

I used the source code instead of the file inside the `_templates` folder because there is no build.xml file code of this version. Instead, I modified and used the build.xml file of version 5.2.

I'm still a bit confused, why is there an `org.xml.sax` package in the source file, but not in the binary? Was it removed after compilation was complete and before being compressed into the release binary?
